assets/xml/objects/object_po_composer.xml

```diff
@@ -1,35 +1,36 @@
 ﻿<Root>
+    <ExternalFile XmlPath="objects/gameplay_keep.xml" OutPath="assets/objects/gameplay_keep/"/>
+
     <File Name="object_po_composer" Segment="6">
```

Comment:
Can you put a comment above the `<File>` node explaining what assets are in here?